airplanedb.py

```diff
@@ -177,4 +177,290 @@ def reset_db(self):
         cursor.execute(create_schedule_table)
         print(('Created new {0} table in {1}').format('SCHEDULE',self.db))
 
+        print(('{0} RESET COMPLETE').format(self.db))
+        cursor.close()
+
+
+    '''
+    function: populate_db
+    description: populate database with test data
+    notes: do this right after reset_db()
+    '''
+    def populate_db(self):
+        db = MySQLdb.connect(host=self.host,
+                                 user=self.user,
+                                 passwd=self.pw,
+                                 db=self.db)
+        cursor = db.cursor()
+
+        ''' insert test customers'''
+        insert_customer_1 = """ INSERT INTO CUSTOMER (C_NAME, C_AGE, C_EMAIL, C_PHONE)
+                            VALUES ('TestCustomer1', 20, 'check@test.com', '6041111111')
+                            """
+
+        insert_customer_2 = """ INSERT INTO CUSTOMER (C_NAME, C_AGE, C_EMAIL, C_PHONE)
+                            VALUES ('TestCustomer2', 30, 'check2@test.com', '6042222222')
+                            """
+
+        insert_customer_3 = """ INSERT INTO CUSTOMER (C_NAME, C_AGE, C_EMAIL, C_PHONE)
+                            VALUES ('TestCustomer3', 40, 'check3@test.com', '6043333333')
+                            """
+
+        insert_customer_4 = """ INSERT INTO CUSTOMER (C_NAME, C_AGE, C_EMAIL, C_PHONE)
+                            VALUES ('TestCustomer4', 50, 'check4@test.com', '6044444444')
+                            """
+
+        try:
+            cursor.execute(insert_customer_1)
+            print(('Created new {0}: {1}').format('CUSTOMER', 'TestCustomer1'))
```

Comment:
Would be worthwhile to stick all these string templates inside of a global strings module like `strings.py` or something and then `import strings` so we don't have to keep C+Ving the same stuff again and again